pkgs/games/anki/fix-paths.patch

```diff
@@ -1,99 +1,105 @@
--else:
--    mplayerCmd = ["mplayer"]
-+mplayerCmd = ["@mplayer@/bin/mplayer"]
- mplayerCmd += ["-really-quiet", "-noautosub"]
- 
+@@ -138,7 +139,7 @@ def cleanupMPV():
  # Mplayer in slave mode
-@@ -220,7 +208,7 @@ class _Recorder(object):
+ ##########################################################################
+
+-mplayerCmd = ["mplayer", "-really-quiet", "-noautosub"]
++mplayerCmd = ["@mplayer@/bin/mplayer", "-really-quiet", "-noautosub"]
+ if isWin:
+     mplayerCmd += ["-ao", "win32"]
+
+@@ -286,7 +287,7 @@ class _Recorder:
          self.encode = encode
          for c in processingChain:
              #print c
 -            if not self.encode and c[0] == 'lame':
 +            if not self.encode and c[0] == lameCmd:
                  continue
              try:
-                 ret = retryWait(subprocess.Popen(c, startupinfo=si))
-diff -Nurp anki-2.0.33.orig/aqt/__init__.py anki-2.0.33/aqt/__init__.py
---- anki-2.0.33.orig/aqt/__init__.py	2015-12-27 11:23:02.338908782 +0100
-+++ anki-2.0.33/aqt/__init__.py	2015-12-27 12:35:03.405565214 +0100
-@@ -107,7 +107,7 @@ def setupLang(pm, app, force=None):
+                 cmd, env = _packagedCmd(c)
+diff --git a/aqt/__init__.py b/aqt/__init__.py
+index dc25057..2e6f7b5 100644
+--- a/aqt/__init__.py
++++ b/aqt/__init__.py
+@@ -143,7 +143,7 @@ def setupLang(pm, app, force=None):
          app.setLayoutDirection(Qt.LeftToRight)
      # qt
      _qtrans = QTranslator()
 -    if _qtrans.load("qt_" + lang, dir):
-+    if _qtrans.load("qt_" + lang, "@qt4@/share/@qt4name@/translations"):
++    if _qtrans.load("qt_" + lang, "@qt5@/share/@qt5name@/translations"):
          app.installTranslator(_qtrans)
- 
+
  # App initialisation
-diff -Nurp anki-2.0.33.orig/oldanki/lang.py anki-2.0.33/oldanki/lang.py
---- anki-2.0.33.orig/oldanki/lang.py	2015-12-27 11:23:02.390909551 +0100
-+++ anki-2.0.33/oldanki/lang.py	2015-12-27 14:05:51.663920453 +0100
-@@ -32,11 +32,7 @@ def ngettext(single, plural, n):
-     return localTranslation().ungettext(single, plural, n)
- 
- def setLang(lang, local=True):
--    base = os.path.dirname(os.path.abspath(__file__))
--    localeDir = os.path.join(base, "locale")
--    if not os.path.exists(localeDir):
--        localeDir = os.path.join(
--            os.path.dirname(sys.argv[0]), "locale")
-+    localeDir = "@anki@/share/locale"
-     trans = gettext.translation('libanki', localeDir,
-                                 languages=[lang],
-                                 fallback=True)
+diff --git a/aqt/mediasrv.py b/aqt/mediasrv.py
+index f1fffb9..ee5b72b 100644
+--- a/aqt/mediasrv.py
++++ b/aqt/mediasrv.py
+@@ -12,16 +12,7 @@ import threading
+
+ # locate web folder in source/binary distribution
+ def _getExportFolder():
+-    # running from source?
+-    srcFolder = os.path.join(os.path.dirname(__file__), "..")
+-    webInSrcFolder = os.path.abspath(os.path.join(srcFolder, "web"))
+-    if os.path.exists(webInSrcFolder):
```

Comment:
This path was not found?

Reply:
I think not... I can't really remember because I did this at least a couple of weeks ago

Reply:
Could be `web` copied next to the `aqt` directory